com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Bob.java

```diff
@@ -672,16 +672,28 @@ private static void mainInternal(String[] args) throws IOException, CompileExcep
         String rootDirectory = getOptionsValue(cmd, 'r', cwd);
         String sourceDirectory = getOptionsValue(cmd, 'i', ".");
 
+        String build_report_json = null;
+        String build_report_html = null;
```

Comment:
We don't have a project here, but we want to have time profiler initialized as soon as possible, so that's why this back compatibility code has been written this way.